### ISSUE TITLE
(Soft-)deprecate `column_map()`?

### ISSUE THREAD

**Issue by jmobrien** [ropensci/qualtRics#284]
`column_map()` uses the old endpoint (e.g. #267), sometimes creates issues (e.g. #283), and has an established replacement (`extract_colmap()`).  Should we go ahead and deprecate it?

If so, should we soft-deprecate w/a warning pointing to the new function, or go ahead and deprecate fully (throwing a similarly-worded error)?  

Leaning towards the latter since I think the older version really doesn't offer anything uniquely useful, so most all users probably want the newer maps.

**Comment by juliasilge** [comment]
Yeah, let's [go straight to `deprecate_warn()`](https://lifecycle.r-lib.org/articles/communicate.html#gradual-deprecation) with this. Good idea.

**Comment by jmobrien** [comment]
Okay, I was using the language inadequately here, but thanks for the link.  `deprecate_warn()` makes sense.
